tests/index-test.js

```diff
@@ -157,7 +157,17 @@ describe('compress plugin', function() {
           'assets/ignore.js',
         ],
         ui: mockUi,
-        project: { name: function() { return 'test-project'; }, require },
```

Comment:
The mock of `project.require` here was actually providing the "full" `require` API, so this was passing in tests.